ovos_gui_plugin_shell_companion/__init__.py

```diff
@@ -44,13 +43,12 @@ def __init__(self, config: dict, bus: MessageBusClient = None,
         LOG.info("OVOS Shell: Initializing")
         super().__init__(config=config, bus=bus, gui=gui,
                          preload_gui=preload_gui, permanent=permanent)
-        self.local_display_config = get_ovos_shell_config()
```

Comment:
since this repo never had a stable release, and this functionality never worked... i'd rather skip this migration step

i'd rather not support at all a non standard file that was introduced sneakily and never worked or had any QA, let's just pretend it never existed...

the file is valid for those using deprecated plugins in ovos-core 0.0.7 only, and again it never worked even then :sweat_smile: